### PR TITLE
release-v1.1: update release references

### PR DESCRIPTION
**Pull request by schristoff**
Signed-off-by: Sarah Christoff <sarah.christoff13@gmail.com>

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -82,8 +82,8 @@ prism_syntax_highlighting = true
 project_home = "https://openservicemesh.io/"
 
 # Version params
-osm_branch = "main"
-osm_version = "v0.11.0"
+osm_branch = "release-v1.1"
+osm_version = "v0.1.1"
```
Comment (jaellio) [inline config.toml:86]:
```suggestion
osm_version = "v1.1.0"
```

File: config.toml [hunk 2]
```diff
@@ -82,8 +82,8 @@ prism_syntax_highlighting = true
 project_home = "https://openservicemesh.io/"
 
 # Version params
-osm_branch = "main"
-osm_version = "v0.11.0"
+osm_branch = "release-v1.1"
+osm_version = ""v1.1.0-rc.1"
```
Comment (nojnhuh) [inline config.toml:86]:
```suggestion
osm_version = "v1.1.0-rc.1"
```

**Review by nojnhuh — CHANGES_REQUESTED** [review]
Let's also update the API reference docs in this PR: https://github.com/openservicemesh/osm/blob/v1.1.0-rc.1/docs/release_guide.md#3-update-api-reference-documentation

File: content/docs/api_reference/config/v1alpha1.md [hunk 3]
```diff
@@ -1,29 +1,23 @@
----
-title: "Config v1alpha1 API Reference"
-description: "Config v1alpha1 API reference documentation."
-type: docs
----
-
```
Comment (nojnhuh) [inline content/docs/api_reference/config/v1alpha1.md:6]: We should keep the header.

Also please generate the docs for config/v1alpha2 and policy/v1alpha1.

**Review by nojnhuh — CHANGES_REQUESTED** [review]

File: content/docs/api_reference/policy/v1alpha1.md [hunk 4]
```diff
@@ -1,29 +1,23 @@
----
-title: "Policy v1alpha1 API Reference"
-description: "Policy v1alpha1 API reference documentation."
-type: docs
----
-
```
Comment (nojnhuh) [inline content/docs/api_reference/policy/v1alpha1.md:6]: Same as before about keeping the header.

And I don't see config/v1alpha2?

Comment (schristoff) [reply content/docs/api_reference/policy/v1alpha1.md:6]: oh did i not git add

Comment (schristoff) [reply content/docs/api_reference/policy/v1alpha1.md]: ok fixed maybe

**Review by nojnhuh — APPROVED** [review]
LGTM. Would you mind making sure all your commits are signed off, either by signing off each individual commit or squashing? I'll hit "squash and merge" either way, so whatever's easiest for you.

**Pull request merged**